pkg/ephemeral-disk-utils/utils_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package ephemeraldiskutils
+
+import (
+	"io/ioutil"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("FileExists", func() {
+	It("recognizes am existing file", func() {
+		Expect(FileExists("/etc/passwd")).To(BeTrue())
+	})
+	It("recognizes non-existing file", func() {
+		Expect(FileExists("no one would ever have this file")).To(BeFalse())
+	})
+})
+var _ = Describe("RemoveFile", func() {
+	It("silently ignores non-existing file", func() {
+		// Ingnoring missing files is typically a sloppy behavior. This test
+		// documents it, not aproves of its usage.
+		Expect(RemoveFile("no one would ever have this file")).To(BeNil())
```

Comment:
Maybe you want to rename it to `ForceRemoveFile` with a document string, to indicate the relationship to `rm -f <file>` or `RemoveFileIfExists`?
Due to mostly dealing with idempotent flows this behaviour is almost always our wanted default behaviour.

Reply:
Renaming is a good idea, but as always, finding a good name is hard. "Force" has the niceness of reminding of `rm -f`, but it is generally a very confusing term - we cannot "force" removal in case there's an IO error or a permission error. I therefor prefer `RemoveFileIfExists`.

Reply:
@rmohr please consider my suggested `RemoveAllAndLogFailure` name.